featurebyte/service/table_facade.py

```diff
@@ -122,12 +122,13 @@ async def update_table_column_cleaning_operations(
 
         columns_info = table.columns_info
         col_name_to_cleaning_operations = {
-            col_clean_op.column_name: col_clean_op for col_clean_op in column_cleaning_operations
+            col_clean_op.column_name: col_clean_op.cleaning_operations
+            for col_clean_op in column_cleaning_operations
         }
         for col_info in columns_info:
             if col_info.name in col_name_to_cleaning_operations:
                 col_info.critical_data_info = CriticalDataInfo(
```

Comment:
Maybe we can consider enforcing strict validation on unexpected fields to prevent similar issues.